src/features/user/Account/EditModal.tsx

```diff
@@ -117,13 +123,13 @@ export const EditModal = ({
 
     if (Object.keys(bodyToSend).length !== 0) {
       try {
-        const res = await putAuthenticatedRequest(
+        const res: ModifyDonations = await putAuthenticatedRequest(
           `/app/subscriptions/${record?.id}?scope=modify`,
           bodyToSend,
           token,
           logoutUser
         );
-        if (res?.status === 'action_required') {
+        if (res?.status !== 'action_required') {
           window.open(res.response.confirmationUrl, '_blank');
```

Comment:
Gives following error
Property 'response' does not exist on type 'ModifyDonations'

Reply:
I'm not sure what response would be received for `status=action_required`. I'm waiting for information from Jorgo.

Reply:
@prachigarg19 Please create an issue for this, and resolve later (once the scenario is replicable)